components/Controls.tsx

```diff
@@ -134,9 +134,8 @@ function SetSpeedLimitButton({ country, maxSpeed, selected, select }: SetSpeedLi
 function BellTone({ bike }: { bike: Bike }) {
     const [currentTone, setCurrentTone] = useState<BellToneEnum | undefined>(undefined)
 
-    // I will try to implement this later.
-    // const obtainFromBike = () => bike.getBellTone().then(setCurrentTone)
-    // useEffect(() => { obtainFromBike() }, [])
+    const obtainFromBike = () => bike.getBellTone().then(setCurrentTone)
+    useEffect(() => { obtainFromBike() }, [])
```

Comment:
This can be simplified
```suggestion
    useEffect(() => { bike.getBellTone().then(setCurrentTone) }, [])
```